packages/node_modules/@webex/widget-call-history/src/utils/dateUtils.ts

```diff
@@ -127,3 +127,37 @@ export function formatDateDDMMYYYY(date: string) {
     year: 'numeric',
   });
 }
+
+/**
+ * @description Handle formatting the date for announce.
+ * @param {string} date The date string to format
+ * @returns {string} A announced date as in "Today", "Yesterday", or "june 10,2024"
+ */
+export const formatDateForAnnouncement = (dateValue: string) => {
```

Comment:
I've reviewed the formatDateForAnnouncement method in the PR and noticed that it currently returns "Today" and "Yesterday" as static strings. This approach does not support localization, which could be an issue for users in different locales.


Fortunately, we have an existing formatDate method that handles localization for "today" and "yesterday" by using d.toRelativeCalendar({}) from the Luxon library, which is designed to support localization.


To ensure consistency and to take advantage of Luxon's localization support, I suggest we update the formatDateForAnnouncement method as follows:

export const formatDateForAnnouncement = (date: string) => {
  const d = getDateTimeFromString(date);

  if (
    d.toISODate() === DateTime.now().toISODate() ||
    d.toISODate() === DateTime.now().minus({ days: 1 }).toISODate()
  ) {
    return d.toRelativeCalendar({});
  }

  return d.toLocaleString({
    month: 'long', // Use 'long' for full month name
    day: 'numeric',
    year: 'numeric', 
  });
}
This updated method will now return localized strings for "today" and "yesterday" and will format other dates in a more accessible manner for announcements.

Reply:
Sure will check for other language also